pkg/virt-operator/util/client.go

```diff
@@ -180,3 +195,11 @@ func remove(users []string, user string) ([]string, bool) {
 	}
 	return newUsers, modified
 }
+
+func IsOnOpenshift(clientset kubecli.KubevirtClient) (bool, error) {
+	_, err := clientset.SecClient().SecurityContextConstraints().List(metav1.ListOptions{})
+	if err != nil && strings.Contains(err.Error(), "the server could not find the requested resource") {
```

Comment:
There's this "ServerResources()" function in the discovery client that allows us to get a list of all the resources supported by the cluster. This is the same thing that 'kubectl api-resources' uses.

That function is vendored in this file.

```
vendor/k8s.io/client-go/discovery/discovery_client.go
```

A method like that might allow us to more accurately determine the scc support.

The concern about parsing the err message is that if that error message changes someday, it breaks our operator.

Reply:
ah nice, didn't know the discovery client. Will use that in a follow up for sure. Thanks for review and merging!